packages/common/src/audius-query/createApi.ts

```diff
@@ -41,12 +44,24 @@ import {
   PerKeyState,
   SliceConfig,
   QueryHookResults,
-  FetchResetAction
+  FetchResetAction,
+  RetryConfig
 } from './types'
 import { capitalize, getKeyFromFetchArgs, selectCommonEntityMap } from './utils'
 
 const { addEntries } = cacheActions
 
+const defaultRetryConfig: RetryConfig = {
+  retries: 3
+}
+
+const isNonRetryableError = (e: unknown) => {
+  if (e instanceof ResponseError) {
+    return [400, 401, 403].includes(e.response.status)
```

Comment:
I'd argue this includes _all_ client errors (400 ≤ status < 500) but I understand the idea around retrying 404s

Reply:
Yep, probably right. Maybe I'll invert this and say if it's between 400-500 and _not_ a 404, it's unrecoverable